src/libs/ReportUtils.ts

```diff
@@ -3193,7 +3209,7 @@ function canEditMoneyRequest(reportAction: OnyxInputOrEntry<ReportAction<typeof
         return false;
     }
 
-    const moneyRequestReportID = originalMessage?.IOUReportID ?? -1;
+    const moneyRequestReportID = originalMessage?.IOUReportID;
 
     if (!moneyRequestReportID) {
```

Comment:
It looks like before `moneyRequestReportID` always was a **truly** value. So this condition never worked. Should I remove it maybe?